pkg/apis/core/validation/shoot.go

```diff
@@ -878,6 +878,20 @@ func ValidateClusterAutoscaler(autoScaler core.ClusterAutoscaler, fldPath *field
 		allErrs = append(allErrs, field.NotSupported(fldPath.Child("expander"), *expander, availableClusterAutoscalerExpanderModes.List()))
 	}
 
+	if ignoreTaints := autoScaler.IgnoreTaints; ignoreTaints != nil {
+
+		supportedVersion := semver.MustParse("1.16")
+		k8sVersion := semver.MustParse(k8sVersion)
```

Comment:
Version checks are typically done this way: https://github.com/gardener/gardener/blob/fdc0953d1eb2e607e5ea1ca43fcf0fad539f844b/pkg/apis/core/validation/shoot.go#L1486-L1488

You might want to adapt :)